### ISSUE TITLE
[4.0.0-beta.6] broke the side navigation

### ISSUE THREAD

**Issue by redwolf2** [ionic-team/ionic#15386]
# Bug Report

**Ionic Info**
```
ionic (Ionic CLI)          : 4.1.1
Ionic Framework            : @ionic/angular 4.0.0-beta.6
@angular-devkit/core       : 0.7.5
@angular-devkit/schematics : 0.7.5
@angular/cli               : 6.1.5
@ionic/ng-toolkit          : 1.0.7
@ionic/schematics-angular  : 1.0.5
```

**Describe the Bug**
The current version doesn't seem to open the side navigation via swipe or the toggle button at all.
Downgrading to @ionic/angular@4.0.0-beta.5 fixes it.

**Related Code**
[menu_problem.zip](https://github.com/ionic-team/ionic/files/2335337/menu_problem.zip)

(This is the default sidenav project with all packages updated to the latest version)

**Update:**

[4.0.0-beta.7] Fixed the opening via toggle button (the hamburger menu). Also closing by swiping works. But swipe to open is still broken.

**Comment by fperez-eypscap** [comment]
i'm having the same problem

**Comment by mburger81** [comment]
@manucorporat 
Yes I have the same problem.

You can also reproduce this in this ways:
- start an empty `sidemenu` project
- update only `@ionic/angular` to `beta6`
- adapt `global.scss` file with new `core.css` import and remove `colors.css` import 
- remove splitpane

now you can reproduce the issue

**Comment by maxgaurav** [comment]
I am facing the same problem on a fresh installation when upgrading to @ionic/angular 4.0.0-beta.6

**Comment by yogeshmayacharya** [comment]
I am also facing the same problem

**Comment by macchie** [comment]
Same here!

**Comment by brandyscarney** [comment]
Thanks for the issue. This has been fixed in master and will be in the `4.0.0-beta.7` which we plan to release later today.

**Comment by redwolf2** [comment]
It is still not solved in [4.0.0-beta.7]. The burger menu button works now, but a swipe still doesn't open it.

**Comment by manucorporat** [comment]
Fixed in master! available next release
